### DISCUSSION TITLE
`extends`  support "," syntax.

### DISCUSSION THREAD

**Discussion by jqhr** [dcampos/nvim-snippy#107] [General]
The `extends` could support multiple filetypes?
For example
```snippets
extends html,javascript,css
```
I have tried this, but not work.
The code must be
```snippets
extends html
extends javascript
extends css
```
For `honza/vim-snippets`, it's vue snippets has support above syntax. 
Could support this feature?

**Comment by dcampos** [comment]
Yes, that syntax should be supported. I opened a issue.
